libcst/_nodes/statement.py

```diff
@@ -1700,6 +1700,8 @@ def get_docstring_impl(
         evaluated_value = val.evaluated_value
     else:
         return None
+    if isinstance(evaluated_value, bytes):
+        return None
```

Comment:
I had to look up the [specs for docstrings](https://peps.python.org/pep-0257/#specification). Looks like they are guaranteed to be string literals (as opposed to bytes literals), so this is fine (although I personally would've written `if not isinstance(evaluated_value, str)`)!